test/spec/modules/userId_spec.js

```diff
@@ -513,6 +513,20 @@ describe('User ID', function () {
       expect(utils.logInfo.args[0][0]).to.exist.and.to.contain('User ID - usersync config updated for 1 submodules');
     });
 
+    it('handles config with name in different case', function () {
+      setSubmoduleRegistry([pubCommonIdSubmodule, unifiedIdSubmodule, id5IdSubmodule, identityLinkSubmodule, merkleIdSubmodule, netIdSubmodule, nextrollIdSubmodule, sharedIdSubmodule, intentIqIdSubmodule, zeotapIdPlusSubmodule, pubProvidedIdSubmodule, criteoIdSubmodule, mwOpenLinkIdSubModule, tapadIdSubmodule, uid2IdSubmodule, admixerIdSubmodule, deepintentDpesSubmodule]);
```

Comment:
I don't think it's necessary to add all submodules here, right? Only one we really need is `criteoIdSubmodule` if that's the one you're testing with. I just don't want id providers to think they have to add their module here. 

Better yet - use a mock id module and no one will feel left out ;-)

Reply:
No worries, I was just following the formatting used by the surrounding tests.  Updated.